content/docs/apps/experimental/restricting-users-to-trusted-ip-ranges.md

```diff
@@ -20,5 +20,60 @@ To fully limit access to your apps and services, you must ensure that all users
 
 ## Restricting access to your own applications
 
-To limit requests to your own applications, you can modify your application logic directly. Alternatively, you can [can follow this example](https://github.com/alphagov/re-paas-ip-safelist-service#quick-demo) within cloud.gov to [create a route service](https://docs.cloudfoundry.org/services/route-services.html#introduction) 
-to block all traffic that does not originate from a trusted IP address range.
+Limiting or restricting access for both ingress and egress traffic from your applications is fairly straightforward through a feature called [Application Security Groups](https://docs.cloudfoundry.org/concepts/asg.html), or ASGs. ASGs support white- and black-listing network traffic from given IPv4 ranges.
+
+### Scoping
```

Comment:
I'd cut this section on scoping. It's not very relevant to customers, and they can read about it in the CF docs if they need to.

Reply:
I only cover scoping so I can specify that we do not deploy platform-scoped ASGs - I think it's worth leaving in there. I did want to specify that we will not accept requests for platform-scoped policies but that seems a bit aggressive for the docs.